server/webapp/WEB-INF/rails/app/assets/javascripts/lib/jquery-pinOnScroll.js

```diff
@@ -87,12 +87,6 @@
       var throttledFixElement   = _.throttle(fixElement, 100);
       var throttledUnFixElement = _.throttle(unFixElement, 100);
 
-      window.setInterval(function () {
-        maybeCallUsingRequestAnimationFrame(function () {
```

Comment:
I believe I'd added this because the pinning would otherwise not work when new content was added to the page (by way of fetching new console logs via ajax).

If it's possible to pin the button by removing the code, seems much better to me.

Reply:
Ah! I see. I'll test out this scenario and see if we can find a way around it.

Reply:
Listening to some existing [events](https://github.com/gocd/gocd/pull/5820/commits/a02fb9ca81c9b3405af07894cb472e0cbcc41c4d) seems to fix this without the need for polling.